examples/siro_sandbox/sandbox_app.py

```diff
@@ -528,6 +534,7 @@ def sim_update(self, dt):
                 obj = json.loads(keyframe_json)
                 assert "keyframe" in obj
                 keyframe_obj = obj["keyframe"]
+                self._messaging_service.add_message_to_keyframe(keyframe_obj)
```

Comment:
I don't like that our "keyframe" term is now ambiguous (it means a gfx-replay keyframe OR it means that PLUS messages). But we can probably refactor this later.